### PR TITLE
Update release notes for 1.4.3 release

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat @rhatdan @baude @haircommander PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+- Fixed a bug where `podman inspect --format '{{.Mounts}}'` would print a hexadecimal address instead of a container's mounts
+- Fixed a bug where rootless Podman would not add an entry to container's `/etc/hosts` files for their own hostname ([#3405](https://github.com/containers/libpod/issues/3405))
+- Fixed a bug where `podman ps --sync` would segfault ([#3411](https://github.com/containers/libpod/issues/3411))
+- Fixed a bug where `podman generate kube` would produce an invalid ports configuration ([#3408](https://github.com/containers/libpod/issues/3408))
+
+### Misc
+- Podman now performs much better on systems with heavy I/O load
+- The `--cgroup-manager` flag to `podman` now shows the correct default setting in help if the default was overridden by `libpod.conf`
+- For backwards compatability, setting `--log-driver=json-file` in `podman run` is now supported as an alias for `--log-driver=k8s-file`. This is considered deprecated, and `json-file` will be moved to a new implementation in several releases ([#3363](https://github.com/containers/libpod/issues/3363))
```
Comment (haircommander) [inline RELEASE_NOTES.md]: suggest rewording `in several releases` to `in the future`. the former seems specifically ambiguous which I find odd

**Comment by haircommander** [comment]
one nit otherwise LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
